pkg/virt-api/rest/subresource.go

```diff
@@ -1259,3 +1263,154 @@ func (app *SubresourceAPIApp) VMIAddVolumeRequestHandler(request *restful.Reques
 func (app *SubresourceAPIApp) VMIRemoveVolumeRequestHandler(request *restful.Request, response *restful.Response) {
 	app.removeVolumeRequestHandler(request, response, true)
 }
+
+func (app *SubresourceAPIApp) SEVFetchCertChainRequestHandler(request *restful.Request, response *restful.Response) {
+	if !app.clusterConfig.WorkloadEncryptionSEVEnabled() {
+		writeError(errors.NewBadRequest(fmt.Sprintf(featureGateDisabledErrFmt, virtconfig.WorkloadEncryptionSEV)), response)
+		return
+	}
```

Comment:
I personally do not like putting error-checking code in helper functions like that. In my view this code is already clear enough. Do you think it would be better to have smth like bellow instead?
```go
if writeErrToResponseIfSevDisabled(response) {
    return
}
```

Reply:
Looks great :) thanks!